test_conformance/math_brute_force/main.cpp

```diff
@@ -140,7 +140,8 @@ int doTest(const char *name)
         const Func *const temp_func = functionList + i;
         if (strcmp(temp_func->name, name) == 0)
         {
-            if (i < gStartTestNumber || i > gEndTestNumber)
+            if ((gStartTestNumber != -1 && i < gStartTestNumber)
```

Comment:
Would it make more sense to track updates to gStartTestNumber and gEndTestNumber via separate flags instead of relying on the initialized values of these variables?

Reply:
LGTM apart from a small comment which may make parsing a little cleaner.

Reply:
This is really just a quick fixup. I do however agree the whole thing could do with a proper cleanup - I mean, those globals and so on are making the code hard to reason about. A proper framework to parse command line options would definitely help too. And the command line interface itself is also non-intuitive: While this is consistent with the documented usage (`-h`), `./math_brute_force 0 0` runs the first test when one might expect it runs no test... This would require a wider discussion, I suppose.